TMS.Net07.Homework.ConverterCurrency/Program.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Threading;
+using System.Globalization;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.ConverterCurrency
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            while (true)
+            {
+                Console.WriteLine("Input exit if you want to close the program\n\n");
+                Console.WriteLine("Input source currency: ");
+                var sourcecurrency = Console.ReadLine().ToUpper();
```

Comment:
Please, use `camelCase` for variables
```suggestion
                var sourceCurrency = Console.ReadLine().ToUpper();
```